.github/workflows/build.yml

```diff
@@ -494,8 +506,33 @@ jobs:
           LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${{ env.INSTALL_APPIMAGE_DIR }}/usr/lib/jvm/java-17-openjdk/lib"
           export LD_LIBRARY_PATH
 
+          chmod +x AppImageUpdate-x86_64.AppImage
+          ./AppImageUpdate-x86_64.AppImage --appimage-extract
+
+          mkdir -p ${{ env.INSTALL_APPIMAGE_DIR }}/usr/optional
+          mkdir -p ${{ env.INSTALL_APPIMAGE_DIR }}/usr/plugins
+
+          cp -r squashfs-root/usr/bin/* ${{ env.INSTALL_APPIMAGE_DIR }}/usr/bin
+          cp -r squashfs-root/usr/lib/* ${{ env.INSTALL_APPIMAGE_DIR }}/usr/lib
+          cp -r squashfs-root/usr/optional/* ${{ env.INSTALL_APPIMAGE_DIR }}/usr/optional
+          cp -r squashfs-root/usr/optional/* ${{ env.INSTALL_APPIMAGE_DIR }}/usr/plugins
+
+          export UPDATE_INFORMATION="gh-releases-zsync|${{ github.repository_owner }}|${{ github.event.repository.name }}|latest|PrismLauncher-Linux-x86_64.AppImage.zsync" 
+
+          if [ '${{ secrets.GPG_PRIVATE_KEY_ID }}' != '' ]; then
+            export SIGN=1
+            export SIGN_KEY=${{ secrets.GPG_PRIVATE_KEY_ID }}
+            mkdir -p ~/.gnupg/
+            printf "$GPG_PRIVATE_KEY" | base64 --decode > ~/.gnupg/private.key
+            gpg --import ~/.gnupg/private.key
+          else
+            echo ":warning: Skipped code signing for Linux AppImage, as gpg key was not present." >> $env:GITHUB_STEP_SUMMARY
```

Comment:
this doesn't seem to work

Reply:
how so?

Reply:
Nothing is printed in https://github.com/PrismLauncher/PrismLauncher/actions/runs/5309725272

Reply:
... interesting. why would echo not work?

Reply:
I'm dumb...
should be fixed now

Reply:
![image](https://github.com/PrismLauncher/PrismLauncher/assets/508861/882e8861-5e7b-4382-9222-1e66aa5c2f88)

Reply:
if a project owner with access would like to generate and add a gpg key for signing proposes that would be great

Reply:
commenting here so i have a reminder in my inbox

Reply:
I have added a key to the secrets.

```
pub   ed25519/0x9C7A2C9B62603299 2023-08-07 [SC] [expires: 2025-08-06]
      Key fingerprint = E1E3 D5B2 1AFF 7275 1BDB  344F 9C7A 2C9B 6260 3299
uid                              Prism Automata <prismlauncher@scrumplex.net>
sub   cv25519/0xEBB5E7E936E12C8A 2023-08-07 [E] [expires: 2025-08-06]
```